pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -180,17 +180,22 @@ func (m *managerImpl) Start() error {
 	if err != nil {
 		return err
 	}
+	ticker := time.NewTicker(dbusReconnectPeriod)
+
 	go func() {
+		defer ticker.Stop()
 		for {
-			if stop != nil {
-				<-stop
-			}
+			select {
+			case <-ticker.C:
```

Comment:
If `m.start` cost `1 second`(anytime longer than `dbusReconnectPeriod`) and then returns in L195,  this `ticker.C` will not block anymore. I don't believe this behavior is expected, because `time.Sleep` here seems to be a `backoff` mechanism before, and we should not break it.

maybe we can do something like calling `Reset` after `m.start`, or using some helper func like `wait.Forever` in `wait` pkg to achieve the same goal.

Reply:
Thanks for your review, this is very useful. It is better to use `time.Timer` instead of `time.Ticker`